src/python/pants/backend/project_info/rules/fast_dependencies.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import Set
+
+from twitter.common.collections import OrderedSet
+
+from pants.base.specs import AscendantAddresses, DescendantAddresses, SingleAddress, Specs
+from pants.build_graph.address import Address
+from pants.engine.console import Console
+from pants.engine.goal import Goal
+from pants.engine.legacy.graph import HydratedTargets, TransitiveHydratedTargets
+from pants.engine.rules import console_rule
+from pants.engine.selectors import Get
+
+
+class Dependencies(Goal):
+  name = 'fast-dependencies'
```

Comment:
Would be good to document what remains to fully remove V1. I think it was

1) Decide if we want to support `--internal-only` and `--external-only`
2) Decide if we care about including for Python 3rd party requirements the specific version numbers, e.g. `pytest==5.2.1` or if it's okay to only have `3rdparty/python:pytest`
3) Ensure comprehensive test coverage

Reply:
I wasn't sure if I should jam that as a comment in the code or open an issue for that. If it's standard to jam in a comment I'll do that.

Reply:
Ticket that you then link to in a comment is even better :)

Reply:
Deprecating internal/external, and just doing internal-always would be appreciated, honestly. It's always been odd to have non-targets mixed in here. If you're able to do that in this PR, that would help us prepare to swap out the existing implementation.

Reply:
Yay that it’s okay to deprecate! Although let’s save that for a dedicated PR.

Reply:
The default behavior of this rule is equivalent to `./pants dependencies --internal-only src/python/pants/backend/project_info/rules:rules` So that works. I'll look into deprecating internal/external on the task in a follow up PR.